core/src/main/java/com/amplifyframework/core/Amplify.java

```diff
@@ -142,7 +144,24 @@ public static void configure(@NonNull final AmplifyConfiguration configuration,
             // Configure User-Agent utility
             UserAgent.configure(configuration.getPlatformVersions());
 
-            DeveloperMenu.singletonInstance(context).enableDeveloperMenu();
+            boolean disableDevMenu = false;
+
+            try {
+                ApplicationInfo ai = context.getPackageManager().getApplicationInfo(
```

Comment:
I believe our checkstyle prevents variable names less than 3 characters, so. maybe renaming this to `applicationInfo`

Reply:
It passed checkstyle but I can change it

Reply:
It is currently configured do dis-allow them on public APIs. But, I don't think we have a check for internal implementation. We could add it? (I personally prefer to use >=3 everywhere FWIW)